test/commands/code-analyzer/run.test.ts

```diff
@@ -19,6 +28,11 @@ describe('`code-analyzer run` tests', () => {
 			receivedActionInput = input;
 			return Promise.resolve();
 		});
+		receivedTelemetryEmissions = [];
+		jest.spyOn(SfCliTelemetryEmitter.prototype, 'emitTelemetry').mockImplementation((source, eventName, data) => {
```

Comment:
I know this isn't the preferred way of doing this sort of test, but it was the best option available.